cosmwasm/ibc-union/app/ucs03-zkgm/src/contract.rs

```diff
@@ -2035,6 +2368,9 @@ pub struct MigrateMsg {
     token_minter_migration: Option<TokenMinterMigration>,
     // Whether to enable or disable rate limiting while migrating.
     rate_limit_disabled: bool,
+    dummy_code_id: Option<u64>,
+    cw_account_code_id: Option<u64>,
+    unbonding_period: Option<u64>,
```

Comment:
can this not be queried? I'm wondering if we can query it on migrate rather than supplying it manually

Reply:
unfortunately no way to query the unbonding period except through stargate query which I want to avoid depending on

Reply:
yeah that's what I was worried about, makes sense

Reply:
can't be queried but we can reuse the undocumented completion_time attribute of the unbond event